app/views/users/email_alerts/destroy.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1><%= _('Email Alerts Disabled') %></h1>
+
+<p>
+  <%= _('Email alerts for things you follow have now been disabled.') %>
+</p>
+
+<p>
+  <%= _('<a href="{{sign_in_link}}">Sign in</a> to get updates on things ' \
```

Comment:
Seeing this when signed in is a little odd, maybe a link to the wall instead for signed in users?

Reply:
Ah yes! Fixed in 220e08c8c.

I did want to take people directly to the wall, but the route requires a `url_name` param, so doing the redirect via signin gets really messy.

I did just try to add a route for `/profile/wall` to go to the current user's wall, but the controller doesn't handle that without modification, so will have to wait until another day.